docs/dg-site.json

```diff
@@ -0,0 +1,46 @@
+{
+  "baseUrl": "",
+  "titlePrefix": "MarkBind",
+  "pages": [
+    {
+      "glob": "**/*.mbd"
+    },
+    {
+      "glob": "*.md"
+    },
+    {
+      "src": "index.md",
+      "searchable": "no"
+    },
+    {
+      "glob": "devGuide/*.md"
+    }
+  ],
+  "headingIndexingLevel": 6,
+  "ignore": [
+    "_markbind",
+    "_site/*",
+    "lib/*",
+    "*.json",
+    "*.md",
+    "*.mbd",
+    "*.mbdf",
+    "*.njk",
+    ".git/*",
+    "*.pptx",
+    "CNAME"
+  ],
+  "plugins" : [
+    "filterTags"
+  ],
+  "pluginsContext" : {
+    "filterTags" : {
+      "tags": ["environment--dg"]
+    }
+  },
+  "deploy": {
+    "message": "Dev Site Update.",
+    "repo": "https://github.com/MarkBind/devdocs.git",
+    "branch": "master"
```

Comment:
Should this be gh-pages instead?

Reply:
I think so. In fact, it defaults to gh-pages, which means we can omit it?

Reply:
Thanks. Updated. I've set it to gh-pages now, just to make it explicit.